data/authorities.json

```diff
@@ -183,6 +183,25 @@
       }
     }
   },
+  "DC": {
+    "city": {
+      "dc-dc-sustainable-energy-utility": {
+        "name": "DC Sustainable Energy Utility",
+        "city": "Washington, DC",
```

Comment:
This value is load-bearing and needs to match the city names that appear in the zips.csv file. I think the name you want is just Washington.

That said, look at the next comment before you make this change since it might make it moot.